components/lcd/esp_lcd_ili9341/idf_component.yml

```diff
@@ -1,4 +1,4 @@
-version: "2.0.0"
+version: "2.0.1"
```

Comment:
Let's not release this version. So no change in version here.

We don't want to release a new version after every change

Reply:
Okay, looks like there has been a change in our version modification principles.